docs/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml

```diff
@@ -1695,12 +1698,58 @@ dfsSSLPrivateKeyFile=/keyfilepath/keyfile</programlisting>Set the <emphasis
           rather it is an option that must be enabled. If that option is
           enabled that can cause the Thor nodes to hang in the manner
           described. If that option is enabled, then you must unset the
-          option, or add the hpcc user to the AllowUsers list. </para>
+          option, or add the hpcc user to the AllowUsers list.</para>
 
-          <para> </para>
+          <para></para>
         </sect3>
       </sect2>
 
+      <sect2>
+        <title>Multiple Roxie Clusters</title>
+
+        <para>You can configure multiple Roxie clusters. When you have
+        multiple Roxie clusters, it is better to use a load balancer behind
```

Comment:
is the load balancer behind the Roxie clusters or are the Roxie clusters behind a load balancer?